docs/general/administration/hardware-selection.md

```diff
@@ -0,0 +1,158 @@
+
+:::
+
+### Low Power Applications
+
+For low power applications, Intel 12th gen or newer Atom CPUs with integrated graphics are recommended.
+
+### Upgrading Existing System for use as a Server
+
+Intel ARC GPUs are recommended for upgrading existing systems for use as a Jellyfin host. If Intel ARC GPUs aren't available, use Nvidia GTX 16 / RTX 20 or newer instead, excluding the GTX 1650.
+
+## Detailed Guide
+
+### CPU
+
+The CPU will be responsible for many tasks, notably:
+
+- Transcoding Audio
+- Decoding video in unsupported codecs
+
+Audio codecs are very lightweight and most CPUs should be able to handle them without issues. Video in unsupported codecs are usually older formats that are easier to decode. Assuming hardware acceleration is properly configured, any modern CPU with 4 threads should be able to handle the workload.
+
+:::caution
+
+Newer codecs can also require software decoding if there are no available hardware decoders. High bitrate HEVC content such as commonly found in 4k HDR can be very demanding even on modern CPUs. Please considering adding a modern GPU with 10bit HEVC decoding support.
+
+:::
+
+#### Integrated Graphics
+
+Integrated graphics can be useful for transcoding video. Please refer to the GPU section for a guide.
+
+#### Resizable BAR
+
+Resizable Bar (ReBAR) is a PCI Express feature that requires support from the platform. This feature is supported on Intel 10th gen or above and AMD Ryzen 3000 Series or above.
+
+When using Intel ARC Graphics, Resizable BAR is recommended. Disabling it will result in a 10% reduction in transcoding performance. However, given how fast the media engine on Intel ARC is, this will not be a problem for the vast majority of users.
+
+### System Memory (RAM)
+
+As a general rule of thumb, at least 8GB is recommended, though significantly smaller amounts may work just fine. If integrated graphics is being used, please consider adding more as integrated graphics usually use part of the system memory as video memory.
+
+### Graphics Cards (GPUs)
+
+With modern systems, the media engines are usually located on the GPUs. Therefore, the available hardware acceleration options are determined by the choice of GPU.
+
+Encoder Quality: Intel > Nvidia > AMD > Apple
+
+Intel graphics is recommended over Nvidia graphics because of the quality. AMD or Apple Silicon are not recommended since they produce poor quality H.264 and H.265(HEVC) output.
+
+:::note A Note on AMD AV1 Encoders
+
+With AV1, AMD has significantly improved the quality of their encoders. However, transcoding to AV1 is currently not supported on Jellyfin, and transcoding to H.264 or H.265(HEVC) is still more likely due to AV1 being in the early stages of adoption.
```

Comment:
To give present accurate description i would rephrase to:
`[...]With jellyfin server version 10.8.10, AV1 codec is not supported[...]`
here instead of below